model/etcd.go

```diff
@@ -14,6 +15,7 @@ type Etcd struct {
 	DisasterRecovery   EtcdDisasterRecovery `yaml:"disasterRecovery,omitempty"`
 	VolumeMounts       []VolumeMount        `yaml:"volumeMounts,omitempty"`
 	EC2Instance        `yaml:",inline"`
+	OPTS               EtcdOpts     `yaml:"opts,omitempty`
```

Comment:
This is very generic - would something like UserSuppliedArgs be more descriptive?

Reply:
will change.